Registry/registry.polrad

```diff
@@ -0,0 +1,61 @@
+
+rconfig   integer sbm_diagnostics          namelist,physics         max_domains    0       rh    "sbm_opt"            "SBM diagnostic opeion, 1: on"
+
+
+# sbm radar variables
+state   real    -                 ikjf sbmradar      1         -     -   -
```

Comment:
@JS-WRF-SBM 
Kobby,
Please line up the columns, these are much easier to inspect that way